src/components/ArticlesList.jsx

```diff
@@ -1,13 +1,13 @@
 import { useEffect, useState } from "react";
-import { getAllArticles } from "../api";
+import { getArticles } from "../api";
 import ArticleCard from "./ArticleCard";
 
 function ArticlesList() {
   const [articles, setArticles] = useState([]);
   const [isLoading, setIsLoading] = useState(true)
 
   useEffect(() => {
-    getAllArticles().then((response) => {
+    getArticles().then((response) => {
```

Comment:
Could have altered this component so we are using params to query by topic? axios should ignore any null values in the params object, maybe come back to this and have a go refactoring for the sake of DRYness as the articles list/topics page seem the same to me otherwise